src/main/resources/api/swagger.yaml

```diff
@@ -165,42 +162,42 @@ definitions:
     properties:
       name:
         type: string
-        description: Unique identifier representing a specific feature.
+        description: A unique identifier representing a specific feature.
       description:
         type: string
-        description: Description of the feature.
+        description: A description of the feature.
       author:
         type: string
-        description: Name of the feature author.
+        description: The name of the feature's author.
       status:
         type: string
-        description: Status of the feature.
+        description: The status of the feature.
         enum:
-          - approved
-          - declined
-          - pending
+        - approved
+        - declined
+        - pending
```

Comment:
I'm not sure that this indentation is correct; it looks like you are moving these items from enum scope into status scope